changes/2910.misc.rst

```diff
@@ -0,0 +1 @@
+Partial OpenFileDialog implementation
```

Comment:
The changelog message should be written in the form of a potential release note for the new feature. In this case, it should be a `.feature` (this isn't a small miscellaneous fix - it's a major new feature), and should read something like:
```suggestion
Android now has support for  OpenFileDialog.
```

Reply:
And here I didn't expect it to be a major new feature. Because I looked at it and saw how a partial implementation of opening files